.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you are reporting a new issue, make sure that we do not have any duplicates
```

Comment:
nit: If the browser can wrap the text automatically, maybe one line is enough here?

Reply:
fixed